webserver/static/js/feverscreen.ts

```diff
@@ -89,8 +91,158 @@ const populateVersionInfo = async (element: HTMLDivElement) => {
 type BoxOffset = 'left' | 'right' | 'top' | 'bottom';
 type FovBox = Record<BoxOffset, number>;
 
+
+class HaarWeakClassifier {
```

Comment:
Could you move all the HaarCascade code to its own file. feverscreen.ts is getting a bit big.

Reply:
Fixed in 3121d8c